### DISCUSSION TITLE
Better Table Editing

### DISCUSSION THREAD

**Discussion by hkunzhe** [alagrede/znote-app#85] [Ideas]
It is an awesome app! Would it be possible to add some table-related actions to the toolbar?

table-related actions (from https://hackmd.io/)
<img width="543" alt="image" src="https://github.com/alagrede/znote-app/assets/30763967/3717abcf-af2c-4671-84b0-fcdaa56d846d">
 
current toolbar
<img width="376" alt="image" src="https://github.com/alagrede/znote-app/assets/30763967/2be5fe5e-57e0-4374-b3f1-7e98b4eb208c">

**Comment by alagrede** [comment]
Yes, I know, working with Markdown table is painful. As valid workaround, I added some actions in the context menu (right click) to convert the table to csv and vice versa to more easily manipulate the data to display in the table.
I am adding your request to the Roadmap.
Thank you for your feedback

Kind regards,
Anthony

**Comment by alagrede** [comment]
Hi @hkunzhe 👋,  

Following up on this discussion, we’ve introduced a WYSIWYG editor that now includes a Table editor. It’s fully compatible with Markdown syntax, making it easy to switch between rich text editing and code-based formatting.  

<img width="508" alt="Screenshot 2024-12-27 at 12 06 10" src="https://github.com/user-attachments/assets/2482d2bc-94f6-447a-b080-246b3e45236c" />  

Let me know if you have any feedback or further questions! 🚀
